src_server/Api.re

```diff
@@ -0,0 +1,34 @@
+open Express;
+
+let dirname =
+  switch ([%bs.node __dirname]) {
+  | Some(d) => d
+  | None => Js.Exn.raiseError("Could not get __dirname")
+  };
+
+let getRefdomainsJson = () => {
+  Node.Fs.readFileAsUtf8Sync(Node.Path.join([|dirname, "Refdomains.json"|]))
+  |> Js.Json.parseExn;
+};
+
+let app = express();
+
+App.disable(app, ~name="x-powered-by");
+
+App.get(app, ~path="/refdomains") @@
+Middleware.from((next, req) =>
+  switch (Request.baseUrl(req)) {
```

Comment:
Is this switch expression a hint for them to do other things? If not, just `Response.sendJson(getRefdomainsJson())` should be sufficient.